consensus/istanbul/core/commit.go

```diff
@@ -149,5 +153,9 @@ func (c *core) acceptCommit(msg *message, src istanbul.Validator) error {
 		return err
 	}
 
+	if c.current.Commits.Size() == 1 {
+		vrankFirstCommitArrivalTimeGauge.Update(int64(vrank.TimeSinceStart()))
```

Comment:
This code doesn't do `nil` checking for vrank. is it okay?

Reply:
fixed in 4b9963cf

Reply:
Also checked all `vrank` references. thanks!